modules/objdetect/misc/python/test/test_charuco_board.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import print_function
+
+import os, tempfile, numpy as np
+
+import sys
+import cv2 as cv
+from tests_common import NewOpenCVTests
+
+sys.path.append(NewOpenCVTests.repoPath+"/../doc/pattern_tools")
```

Comment:
Please don't mess modules (bindings) tests and tests for support tools / apps.
That break many things and usually required extra dependencies (e.g. some platforms may not have them at all).

For testing tools / apps use dedicated tests near that tools.